go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/fluxcd/flux2
 go 1.16
 
 require (
+	github.com/Azure/go-autorest/autorest v0.11.18 // indirect
```

Comment:
Can you undo these? I see no code changes in the CLI so I think we don't need any of these packages.